### DISCUSSION TITLE
Remix deployment on Vercel started failing today

### DISCUSSION THREAD

**Discussion by pbilyk** [vercel/community#1547] [Help] [answered]
### Summary

Hello.

I have an app that uses remix and remix-i18next package. And starting today we are seeing the error that indicates that the app is not sending headers the same way it did before.

Is this intentional, and I should just wait until the package author fixes this on their side?Or is this some bug on vercel side?

This is the link to the related issue on the package side for more details: https://github.com/sergiodxa/remix-i18next/issues/117

**Comment by heozeop** [comment]
I faced same problem and fixed with config vercel cli version to 28.15.3
add enviroment variable `VERCEL_CLI_VERSION` to `vercel@28.15.3`.
Well, it is temporal solution though.

**Comment by cill-i-am** [comment]
Probably same issue i'm facing: https://github.com/orgs/vercel/discussions/1549

**Comment by amyegan** [comment]
Thanks for reaching out. I'm sharing this info with the engineering team 🙏

**Comment by swarnava** [comment, marked as answer]
This is fixed on CLI v28.16.2.
